src/time_zone_format.cc

```diff
@@ -31,27 +31,43 @@
 #if !HAS_STRPTIME
 #include <iomanip>
 #include <sstream>
+#include <get_time.h>
```

Comment:
Can we make the include conditional on being compiled with MinGW?  Just to minimise side-effects...